qual_id/pattern.py

```diff
@@ -19,6 +20,8 @@ class Pattern:
       "game":Game(),
       "geography":Geography(),
       "instrument": Instrument()
```

Comment:
```suggestion
      "instrument": Instrument(),
```
You missed a comma @mephi007 
Easy to do, especially in python 🙂